regression/contracts/quantifiers-nested-05/main.c

```diff
@@ -1,8 +1,14 @@
 // clang-format off
-int f1(int *arr) __CPROVER_requires(!__CPROVER_forall {
-    int i;
-    (0 <= i && i < 10) ==> arr[i] == 0
-  }) __CPROVER_ensures(__CPROVER_return_value == 0)
+int f1(int *arr)
+  __CPROVER_requires(
+    !__CPROVER_forall {
+      int i;
+      (0 <= i && i < 10) ==> arr[i] == 0
+    }
```

Comment:
It'd be nice to prove equivalence of
```
(!__CPROVER_forall(P)) == (__CPROVER_exists(!P))
```

Reply:
Perhaps out of scope for this PR.

Reply:
You're right, we haven't tested these two quantifiers together. Each regression test currently is checking either universal or existential quantification, but not both.

I would add it in another iteration. I also plan to debug the quick-sort regression test that's marked as a KNOWNBUG.